src/main/java/com/couchbase/lite/support/RemoteRequest.java

```diff
@@ -267,10 +267,11 @@ protected void preemptivelySetAuthCredentials(HttpClient httpClient) {
         if (userInfo != null) {
             if (userInfo.contains(":") && !userInfo.trim().equals(":")) {
                 String[] userInfoElements = userInfo.split(":");
-                String username = isUrlBasedUserInfo ? URIUtils.decode(userInfoElements[0]): userInfoElements[0];
-                String password = isUrlBasedUserInfo ? URIUtils.decode(userInfoElements[1]): userInfoElements[1];
+                String username = isUrlBasedUserInfo ? URIUtils.decode(userInfoElements[0]) : userInfoElements[0];
+                String password = "";
```

Comment:
I believe @hideki already checked if it should be an empty string instead of null value.